index.html

```diff
@@ -21,237 +21,218 @@ <h1>The open source multicloud control plane.</h1>
-          <div class="name">Sid Sijbrandij</div>
-          <div class="title">CEO and Co-Founder of GitLab</div>
-          <a href="https://gitlab.com"><img src="{{ "/images/gitlab.svg" | relative_url }}"/></a>
+          <div class="exec-img">
+            <img srcset="./images/sid.jpg 1x, ./images/sid@2x.jpg 2x, ./images/sid@3x.jpg 3x" alt="Sid Sijbrandij" />
+          </div>
+          <div class="exec-info">
+            <div class="name">Sid Sijbrandij</div>
+            <div class="title">CEO and Co-Founder of GitLab</div>
+          </div>          
         </div>
       </div>
-      <div class="testimonial col-4_md-6_sm-12">
+      <div class="testimonial col-6_md-12">
         <h3>
-          &ldquo;We share the vision of having truly cloud agnostic, managed
-          services and Crossplane is an important step forward in this
-          direction&rdquo;
+          &ldquo;We share the vision of having truly cloud-agnostic managed
+          services.&rdquo;
         </h3>
         <p>
           &ldquo;Crossplane will make it possible for organizations to take
           deeper ownership of where they want to run their services and how they
           manage their application data. Kudos to the Upbound team for
           architecting and delivering this critical, open source and community
-          driven component of the cloud neutral future.&rdquo;
+          driven component of the cloud-neutral future.&rdquo;
         </p>
         <div class="signature">
-          <div class="name">Spencer Kimball</div>
-          <div class="title">CEO and Founder of Cockroach Labs</div>
-          <a href="https://www.cockroachlabs.com/"
-            ><img src="{{ "/images/cockroachlabs.svg" | relative_url }}"
-          /></a>
+          <div class="exec-img">
+            <img srcset="./images/spencer.jpg 1x, ./images/spencer@2x.jpg 2x, ./images/spencer@3x.jpg 3x" alt="Spencer Kimball" />
+          </div>
+          <div class="exec-info">
+            <div class="name">Spencer Kimball</div>
+            <div class="title">CEO and Founder of Cockroach Labs</div>
+          </div>          
         </div>
       </div>
-      <div class="testimonial col-4_md-6_sm-12">
+      <div class="testimonial col-6_md-12">
+        <h3>
+          &ldquo;Crossplane has the ability to be at the heart of tomorrow’s multicloud deployments.&rdquo;
+        </h3>
+        <p>
+          &ldquo;It's exciting to see open cloud standards like Crossplane emerge, that will allow new multicloud technologies like
+          YugabyteDB to take advantage of the broader cloud ecosystem, on-premise and in-public cloud. Future work with
+          Crossplane.io and Rook.io is a natural fit with the Kubernetes ecosystem.&rdquo;
+        </p>
+        <div class="signature">
+          <div class="exec-img">
+            <img srcset="./images/karthik.jpg 1x, ./images/karthik@2x.jpg 2x, ./images/karthik@3x.jpg 3x" alt="Karthik Ranganathan" />
```

Comment:
I believe you should still have a `src` along with the `srcset` and all of these should use the `relative_url` helper